flycheck-hledger.el

```diff
@@ -77,39 +77,39 @@ https://hledger.org/hledger.html#check."
 
    ;; hledger 1.26 assertions (:LINE:COL)
    (error
-    bol "hledger: Error: balance assertion: " (file-name (minimal-match (one-or-more (not ":")))) ":" line ":" column "\n"
-    (message (one-or-more bol (zero-or-more nonl) "\n")))
+    bol "hledger" (optional ".exe") ": Error: balance assertion: " (file-name) ":" line ":" column line-end
+    (message (one-or-more "\n" (zero-or-more nonl) line-end)))
```

Comment:
it feels wrong to me that the `\n` character from the first line is now part of the message. Can we exclude it with, e.g.,


```emacs
"\n"
(message (one-or-more (zero-or-more nonl) line-end))
```

?

Reply:
I've now seen that this is done in the final commit. Is there a way to organize the commits so they make sense individually?